Sources/Workspace/Destination.swift

```diff
@@ -94,7 +97,7 @@ public struct Destination {
       #if os(macOS)
         // Get the SDK.
         let sdkPath: AbsolutePath
-        if let value = lookupExecutablePath(filename: getenv("SYSROOT")) {
+        if let value = lookupExecutablePath(filename: getenv("SDKROOT")) {
```

Comment:
We'll need to support for env otherwise we risk breaking existing clients

Reply:
Such as?